README.md

```diff
@@ -8,47 +8,46 @@ GraphQL interface on top of the [Devise Token Auth](https://github.com/lynndylan
 ## Table of Contents
 
 <!--ts-->
-* [GraphqlDevise](#graphqldevise)
-   * [Table of Contents](#table-of-contents)
-   * [Introduction](#introduction)
-   * [Installation](#installation)
-      * [Running the Generator](#running-the-generator)
-         * [Mounting the Schema in a Separate Route](#mounting-the-schema-in-a-separate-route)
+   * [GraphqlDevise](#graphqldevise)
```

Comment:
Indentation changed because you are not using the latest version of the TOC generator